wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -513,10 +513,10 @@ def icon(name=None, class_name='icon', title=None):
     Usage:
         {% load wagtailadmin_tags %}
         ...
-        {% icon name="cogs" classname="icon--red" title="Settings" %}
+        {% icon name="cogs" class_name="icon--red" title="Settings" %}
```

Comment:
Hmm… this is a good catch, aligning the mentions in this docstring with the actual argument name – but I wonder if we should actually use `classname` for this argument. It seems like it's more common in the codebase to use `classname` when referring to HTML/CSS classes, and use `class_name` when referring to Python classes.

Reply:
I don't know. But the naming is not a decision made in this pr. Changing this will touch a lot of unrelated code.

I'll add this to the overview ticket #6107 and if needed, we can change it all in one dedicated pr.